tests/common.rs

```diff
@@ -459,6 +462,36 @@ pub fn compare_outputs(
                     .collect(),
                 debug_name: ty.debug_name.as_deref().map(String::from),
             },
+            CoreTypeConcrete::SquashedFelt252Dict(info) => JitValue::Felt252Dict {
+                value: info
+                    .info
+                    .long_id
+                    .generic_args
+                    .iter()
+                    .map(|generic_arg| {
+                        let (key, value) = values.split_first().unwrap();
+
+                        (
+                            Felt::from_bytes_le(
+                                &memory[key.to_usize().unwrap()]
+                                    .clone()
+                                    .unwrap()
+                                    .to_le_bytes(),
+                            ),
+                            match generic_arg {
+                                cairo_lang_sierra::program::GenericArg::Type(ty) => {
+                                    map_vm_values(size_cache, registry, memory, value, ty)
```

Comment:
@edg-l Here I think I put the wrong `value` argument or maybe I missed something else, can I have a bit of guidance on this topic? I think my reconstruction of the dict is wrong